core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -2925,6 +2933,83 @@ private Expression normalize(SqlTypeName typeName, Expression e) {
       }
     }
   }
+
+  /** Implements CLASSIFIER MR Function */
+  private static class ClassifierImplementor implements MatchImplementor {
+    @Override public Expression implement(RexToLixTranslator translator, RexCall call,
+        ParameterExpression row, ParameterExpression rows,
+        ParameterExpression symbols, ParameterExpression i) {
+      return Types.castIfNecessary(String.class,
+          Expressions.call(
+              symbols,
+              Types.lookupMethod(List.class, "get", int.class), i));
+    }
+  }
+
+  /** Implements LAST MR Function */
+  private static class LastImplementor implements MatchImplementor {
+    @Override public Expression implement(RexToLixTranslator translator, RexCall call,
+        ParameterExpression row, ParameterExpression rows,
+        ParameterExpression symbols, ParameterExpression i) {
+      final RexNode node = call.getOperands().get(0);
+
+      final String alpha = ((RexPatternFieldRef) call.getOperands().get(0)).getAlpha();
+
+      final BinaryExpression lastIndex = Expressions.subtract(
```

Comment:
@JulianFeinauer , this local variable is unused. Could you please check if it can be removed? On the other hand, it might mean there's a bug.